server/store/datastore/migration/migration.go

```diff
@@ -51,7 +51,6 @@ var allBeans = []interface{}{
 	new(model.Pipeline),
 	new(model.PipelineConfig),
 	new(model.Config),
-	new(model.File),
```

Comment:
should we drop that table too?

Reply:
Yes, I will create a migration. It is empty for all system as it wasn't used anyways.

Reply:
done